docs/conf/atlantic/2024/writing-day.rst

```diff
@@ -152,3 +152,42 @@ Agenda:
 - Review the `Meetup FAQ <https://www.writethedocs.org/organizer-guide/meetups/faq-meetups/>`__
 
 I recommend contributing to this project using the built in `GitHub web-based editor <https://docs.github.com/en/codespaces/the-githubdev-web-based-editor>`__. Mostly because I am not a Git or GitHub expert.
+
+Audinux - Plan and Publish a User Guide (All day)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Join project organizer, Hank Lee, during Writing Day.
+
+Plan and Publish User Guide for Audinux, a repository for FOSS music applications and plugins
+
+`Audinux <https://audinux.github.io/>`__  is a repository for FOSS music applications and plugins. We want to empower creative people and help them use Audinux to it's full potential! To do that, we need documentarians to help us create a user guide.
+
+Our Writing Day goal is to work together and create a user guide for Audinux. Creating a user guide will positively impact our users and encourage new users.
+
+Our plan is to create new documentation pages within the 'User Guide' templates, check out our `project repository <https://github.com/audinux/audinux.github.io/tree/main/pages/user_guide)>`__. This means you can contribute using the GitHub web editor or using whatever Git-workflow from your favorite local writing setup and text editor.
```

Comment:
check out our `project repository <https://github.com/audinux/audinux.github.io/tree/main/pages/user_guide

// could you remove ) at the end of sentence? Thanks.